packages/server/socketHandlers/handleConnect.ts

```diff
@@ -37,6 +37,11 @@ mutation ConnectSocket {
 
 const handleConnect = async (connectionContext: ConnectionContext) => {
   const {authToken, ip, id: socketId} = connectionContext
+  const {rol} = authToken
+  if (rol === 'impersonate') {
+    connectionContext.ready()
+    return null
```

Comment:
Is the reason we don't want to run `publishInternalGQL` because we don't need subscriptions when impersonating?

Reply:
@nickoferrall that would call the `connectSocket` mutation, which would make the user appear online.
we don't want that 🤫